code/style.css

```diff
@@ -1,3 +1,62 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #EEE;
+  font-family: 'Source Code Pro', monospace;
+}
+
+.profile-img {
+  border-radius: 50%;
+  width: 10rem;
+  margin: 1rem;
+}
+
+.user-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.repo-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: stretch;
+  gap: 2rem;
+  padding: 2rem;
+}
+
+.repo-name {
+  background-color: #FFF;
+  padding: 1rem;
+  width: 20rem;
+}
```

Comment:
I noticed on your mobile version that the .repo-name containers weren't centered horizontally. This can be fixed by adding the property `align-self: center;` to the .repo-name class. When you then expand to a larger viewport, you can change that (because if you don't, it "squishes" some of the containers so they aren't the same size) to `align-self: auto;` by adding a bit of code to line 55. (I'll add it below).